selfdrive/ui/ui.h

```diff
@@ -25,6 +25,17 @@ typedef cereal::CarControl::HUDControl::AudibleAlert AudibleAlert;
 
 const mat3 DEFAULT_CALIBRATION = {{ 0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 1.0, 0.0, 0.0 }};
 
+const int FACE_KPTS_SIZE = 31;
```

Comment:
this shouldn't be hardcoded

Reply:
yeah actually this was for something else, guess can remove